### PR TITLE
Added Ubuntu 14.10

### PR DESCRIPTION
**Pull request by codychamberlain**
Based on testing of DigitalOcean image added Ubuntu 14.10 platform. Had to add python-psutil to CORE dependency list as this is apparently no longer installed by default (at least by DigitalOcean)

### CODE REVIEW

**Comment by andresriancho** [comment]
psutil requirement was a mistake from my side, I've corrected it with https://github.com/andresriancho/w3af/commit/e6914ce01f3f417330edfc6480bfcdbc3edac193

File: w3af/core/controllers/dependency_check/platforms/ubuntu1410.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+"""
+import subprocess
+import platform
+
+from .base_platform import Platform
+from ..requirements import CORE, GUI
+
+
+class Ubuntu1410(Platform):
```
Comment (andresriancho) [inline w3af/core/controllers/dependency_check/platforms/ubuntu1410.py]: Why don't you inherit from `Ubuntu1404`? That would make the `Ubuntu1410` class really small, avoid code duplication at `os_package_is_installed`, etc

**Comment by andresriancho** [comment]
When sending the corrected PR, if possible, re-base it on top of https://github.com/andresriancho/w3af/commit/e6914ce01f3f417330edfc6480bfcdbc3edac193

File: w3af/core/controllers/dependency_check/platforms/ubuntu1410.py (hunk 1 above)
Comment (codychamberlain) [inline w3af/core/controllers/dependency_check/platforms/ubuntu1410.py]: i tried inheriting the class originally but when i needed to include psutil i couldn't override the CORE_SYSTEM_PACKAGE list. now that psutil is removed i'll go back to inheriting the ubuntu class. agreed!

**Pull request closed without merging**